featuretools/utils/gen_utils.py

```diff
@@ -112,15 +113,15 @@ def check_schema_version(cls, cls_type):
             if c_num > s_num:
                 break
             elif c_num < s_num:
-                raise RuntimeError(error_text_upgrade)
+                warnings.warn(error_text_upgrade)
```

Comment:
We should add a `break` after this warning so that we do not continue to iterate after the warning